Source/WebCore/PlatformPlayStation.cmake

```diff
@@ -69,7 +69,6 @@ list(APPEND WebCore_SOURCES
 
 list(APPEND WebCore_USER_AGENT_STYLE_SHEETS
     ${WEBCORE_DIR}/Modules/mediacontrols/mediaControlsBase.css
-    ${WEBCORE_DIR}/css/mediaControls.css
```

Comment:
Why was this change made? It broke the PlayStation build.

Reply:
Reverting this particular change in https://github.com/WebKit/WebKit/pull/18147.

Reply:
Oops. It's my bad. I though https://github.com/WebKit/WebKit/pull/17355 already landed.

Reply:
Sorry, I missed that, I though Adwaita was already used as well.
@fujii @rkirsling : I'd appreciate your review on this follow-up PR too: https://github.com/WebKit/WebKit/pull/18153